docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
```

Comment:
New sentence, new line.

```suggestion
If you are interested in contributing but unsure how to begin, we recommend to start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT). The developers there, which include the team that originally developed KILT as well as the wider KILT community, can:
```

Reply:
We have to use active developers when we refer the KILT community.